coalib/bearlib/languages/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .Language import Languages
 
 from .definitions.Unknown import Unknown
+from .definitions.Bash import Bash
```

Comment:
Why are you adding ```Bash``` here? I don't think it's relevant here. IMO it's from  d5c0b79  which was earlier into this PR. Please check this.

Reply:
It is done by me. When i sync my local repo with remote they are added automatically.
Even I did not know the meaning of bash then when i added this in this file.
And also now you open the coalib/bearlib/languages/__init__.py on github of coala, these all are already there. https://github.com/coala/coala/blob/master/coalib/bearlib/languages/__init__.py  SEE IN THIS LINK BASH IS ALREADY THERE.

Reply:
I think that the github is comparing with old coala files.